### PR TITLE
Benchmarks turnover: adding in actual values

### PR DESCRIPTION
**Pull request by g-m-taylor**
#### Tests
Does this PR include tests for the changes introduced?
- [x] Yes
- [x] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: server/routes/establishments/benchmarks/index.js [hunk 1]
```diff
@@ -69,6 +70,41 @@ const pay = async (establishmentId) => {
...
+        percentOfPermTemp = (leavers / permTemptCount);
+      }else{
+        stateMessage = 'check-data';
+      }
+    } else{
+      stateMessage = 'no-data';
+    }
+  }else{
+    stateMessage = 'no-workers';
+  }
```
Comment (chadtomkiss-madetech) [inline server/routes/establishments/benchmarks/index.js:94]: I think there's too many `if/else` branches here and that `percentOfPermTemp` calculation is in the deepest nest but is actually the main purpose of the function.

There's a good example on using guard clauses to tackle this problem here: https://refactoring.guru/replace-nested-conditional-with-guard-clauses

Comment (g-m-taylor) [reply server/routes/establishments/benchmarks/index.js]: Ok, I had to create an other function to do this, but have a look and see if this is what you mean?

File: server/routes/establishments/benchmarks/index.js [hunk 2]
```diff
@@ -70,28 +70,25 @@ const pay = async (establishmentId) => {
...
+  if (!establishment || establishment.NumberOfStaffValue === 0 ||
+    workerCount !== establishment.NumberOfStaffValue) {
+    return {percentOfPermTemp: 0, stateMessage: 'no-workers'};
+  }
+  const permTemptCount = await models.worker.permAndTempCountForEstablishment(establishmentId);
+  const leavers = await models.establishmentJobs.leaversForEstablishment(establishmentId);
+  if (establishment.LeaversValue !== 'With Jobs') {
+    return {percentOfPermTemp: 0, stateMessage: 'no-data'};
+  }
+  if ((leavers / permTemptCount) > 9.95) {
```
Comment (chadtomkiss-madetech) [inline server/routes/establishments/benchmarks/index.js:85]: Duplication of `(leavers / permTemptCount)` for both the `if` and the `percentOfPermTemp` calculation - could use a variable.

File: server/routes/establishments/benchmarks/index.js [hunk 3]
```diff
@@ -70,28 +70,25 @@ const pay = async (establishmentId) => {
...
-    stateMessage = 'no-workers';
+  if (!establishment || establishment.NumberOfStaffValue === 0 ||
+    workerCount !== establishment.NumberOfStaffValue) {
+    return {percentOfPermTemp: 0, stateMessage: 'no-workers'};
+  }
+  const permTemptCount = await models.worker.permAndTempCountForEstablishment(establishmentId);
+  const leavers = await models.establishmentJobs.leaversForEstablishment(establishmentId);
+  if (establishment.LeaversValue !== 'With Jobs') {
+    return {percentOfPermTemp: 0, stateMessage: 'no-data'};
+  }
```
Comment (chadtomkiss-madetech) [inline server/routes/establishments/benchmarks/index.js:84]: This `if` is right between the consts and where the consts are used - if you move this `if` above the consts you'll get all of your early returns together and then the main flow is the calculation 👍

**Pull request merged**